test/androidTest/java/org/thoughtcrime/securesms/espresso/ConversationSettingsHelper.java

```diff
@@ -16,10 +16,10 @@
 import static android.support.test.espresso.matcher.ViewMatchers.withText;
 import static org.thoughtcrime.securesms.espresso.ViewActions.waitFor;
 
-public class PreferencesHelper extends BaseHelper<PreferencesHelper>{
-    public PreferencesHelper(HelperSecret s) {}
+public class ConversationSettingsHelper extends BaseHelper<ConversationSettingsHelper> {
```

Comment:
is the return type refactoring necessary @shelbaz ?? @g-harel  can you check this

Reply:
What do you mean type refactoring, are you talking about the generic's type?

Reply:
I think other tests are dependent on this function too and now by changing the return type, the other tests will not behave properly, not sure tho. just wanted to bring that up :)